src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -152,8 +162,13 @@ public Response startMonitor(
         session.save(sessionModel);
         GTFSDB.commitAndCloseSession(session);
 
-        //Extract the Url and gtfsId to start the background process
-        startBackgroundTask(gtfsRtFeed, updateInterval);
+        if(feedSession == null) {
```

Comment:
>So I guess simply passing the startTime is the only necessary change? (the ones I made in the monitoring.js file)

Yes, I think this is the primary improvement in this PR - if you could remove the other changes that would be great (modulo the `v1.0.1`).  However, I'm not sure I'm convinced that this alone fixes #215.  I'd like to have @Suryakandukoori dig in a little more before merging this PR, although he's tied up with some other tasks this week.